### PR TITLE
File generation

### PR DESCRIPTION
**Pull request by andrewahn-ubc**
Added JSON file generation functionality and fully tested the code. I pulled right before opening the PR.

### CODE REVIEW

File: i18nilize/src/internationalize/generate_file.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+			{
+				"language": "Spanish",
+				"hello": "Hola",
+			},
+		]
+    }
+
+    # transforms the dictionary object above into a JSON object
+    json_object = json.dumps(file_content, indent=4)
+    create_json(json_object)
```
Comment (kevinrczhang) [inline i18nilize/src/internationalize/generate_file.py:24]: Add this to helpers as well. Makes me uncomfortable to do this but let's stick it into helpers.py for now

**Review by kevinrczhang — CHANGES_REQUESTED** [review]
just some minor changes. This is good

File: i18nilize/src/internationalize/generate_file.py (hunk 1 above)
Comment (andrewahn-ubc) [reply i18nilize/src/internationalize/generate_file.py]: Makes sense to me - done!

**Review by kevinrczhang — APPROVED** [review]
:shipit:

**Review by hriday330 — APPROVED** [review]

**Comment by hriday330** [comment]
Please resolve the helpers.py merge conflict

**Pull request merged**